android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -65,4 +66,113 @@ internal class CommonKtTests {
             setProxyURLString("this is not a url")
         }
     }
+
+    @Test
+    fun `calling canMakePayments correctly passes through to Purchases`() {
```

Comment:
This test is the same as `calling canMakePayments correctly passes through to Purchases` right? The latter is actually more complete